app/actions/ForumActions.ts

```diff
@@ -0,0 +1,170 @@
+import { forumSchema, threadSchema } from 'app/reducers';
+import { Forum, Thread } from './ActionTypes';
+import callAPI from './callAPI';
+import type { ID } from 'app/store/models';
+import type {
+  CreateForum,
+  CreateThread,
+  DetailedForum,
+  DetailedThread,
+  PublicForum,
+  PublicThread,
+  UpdateForum,
+  UpdateThread,
+} from 'app/store/models/Forum';
+
+export function fetchForums() {
+  return callAPI<PublicForum[]>({
+    types: Forum.FETCH_ALL,
+    endpoint: '/forums/',
+    schema: [forumSchema],
+    method: 'GET',
+    meta: {
+      errorMessage: 'Henting av forum feilet',
+    },
+    propagateError: true,
+    requiresAuthentication: true,
+  });
+}
+
+export function fetchForum(forumId: ID) {
+  return callAPI<DetailedForum>({
+    types: Forum.FETCH,
+    endpoint: `/forums/${forumId}/`,
+    schema: forumSchema,
+    meta: {
+      errorMessage: 'Henting av forum feilet',
+    },
+    propagateError: true,
+    requiresAuthentication: true,
+  });
+}
+
+export function createForum(forum: CreateForum) {
+  return callAPI<DetailedForum>({
+    types: Forum.CREATE,
+    endpoint: '/forums/',
+    method: 'POST',
+    body: forum,
+    schema: forumSchema,
+    meta: {
+      errorMessage: 'Opprettelse av forum feilet',
+    },
```

Comment:
Is there any reason why you only added `requiresAuthentication` to the first two endpoints?